recipes/shiver/meta.yaml

```diff
@@ -9,25 +9,26 @@ source:
   patches:
     - 0001-changed-tools-to-shiver_tools.patch
     - 0002-changed-python2-to-python.patch
-    - 0003-added-fastaq-installation-on-demand.patch
-    - 0004-fixed-pyfastaq-installation.patch
+    - 0003-added-fastaq-installation-on-demand.patch  # [py27]
+    - 0004-fixed-pyfastaq-installation.patch  # [py27]
 
 build:
-  skip: True # [osx or not py27]
-  number: 1
+  skip: True  # [osx]
+  number: 3
```

Comment:
number: 2

Reply:
won't lowering the build number now, after a higher one has been uploaded, not work because the lower-numbered package will get masked by a higher-numbered one?